### PR TITLE
Fix #1176: Add blog post for 5.2.5 release

### LINKED ISSUE #1176: Create blog post for 5.2.5 release
The 5.2.5 release needs a blog post outlining new features and fixes.

### PR DESCRIPTION
**Pull request by tst-cjeffries**
Add new blog post.

### CODE REVIEW

**Comment by tst-nfarah** [comment]
let's create an issue and link it in this PR

**Comment by tst-cjeffries** [comment]
Issue was added here: #1176

**Comment by tst-nfarah** [comment]
Is there a github web page where one can look at the formatted html ?

**Comment by tst-cjeffries** [comment]
Give me a minute, and ill get it up and running on github pages.

**Comment by tst-cjeffries** [comment]
HTML is viewable here: https://tst-cjeffries.github.io/earthenterprise/blog.html

File: docs/_posts/2019-01-23-525-release.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: blogpost
+title:  "5.2.5 Release Announcement"
+section_id: blog
+date:   2019-01-23 12:00:00
```
Comment (tst-nfarah) [inline docs/_posts/2019-01-23-525-release.md]: update date to today: 2019-01-24

File: docs/_posts/2019-01-23-525-release.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+**New command line publishing options.** Users can now enable POI search, enhanced search, and WMS when publishing a database from the command line.
+
+**Packets can now be dumped from portable globes.** The command line utility <code>geglxinfo</code> can now extract packets from 3D portable globes in order to explore their contents.
+
+**New cutter advanced option: Continue past empty levels.** When cutting from a 2D map, the “Continue past empty levels” option forces the cutter to continue building the portable map within the polygon region down to the region level, even if the GEE server thinks there is no intermediate imagery to display.
+
+**Installation scripts can no longer be run when the GEE packages are installed.** The install and uninstall scripts will now check if Open GEE has been installed with a package manager before continuing, and will not make any changes if the packages are already installed.
+
+To download this release see the [release page](https://github.com/google/earthenterprise/releases/tag/5.2.5-714.41). The full release notes can also be found [here](http://www.opengee.org/geedocs/answer/7160006.html).
```
Comment (tst-nfarah) [inline docs/_posts/2019-01-23-525-release.md]: use https instead of http

Comment (tst-cjeffries) [reply docs/_posts/2019-01-23-525-release.md]: Done.

**Pull request merged**